### PR TITLE
RN-599 Add entity title in pdf file header

### PR DESCRIPTION
**Pull request by billli0**
### Issue #:
RN-599

### CODE REVIEW

File: packages/lesmis/src/components/DashboardExportModal/components/Header.js [hunk 1]
```diff
@@ -14,26 +15,33 @@ const FlexColumn = styled(BaseFlexColumn)`
...
-  line-height: 140%;
-  text-transform: capitalize;
+const SubHeading = styled(BaseHeading)`
+  font-size: 17px;
 `;
 
 const Header = ({ dashboardLabel, useYearSelector, selectedYear }) => {
+  const { entityCode } = useUrlParams();
+  const { data: entityData } = useEntityData(entityCode);
+  const entityName = entityData?.name ? entityData?.name : '';
```
Comment (rohan-bes) [inline packages/lesmis/src/components/DashboardExportModal/components/Header.js:37]:
```suggestion
  const entityName = entityData?.name || '';
```
Slightly simpler, but I don't really mind 👍

File: packages/lesmis/src/components/DashboardExportModal/components/DashboardNameSubHeader.js [hunk 2]
```diff
@@ -1,16 +0,0 @@
...
-    <Svg width="497" height="3" viewBox="0 0 497 3">
-      <Text x="0" y="15">
-        {dashboardName}
-      </Text>
-      <Line x1="0" y1="20" x2="497" y2="20" stroke="#D13333" strokeWidth="3" />
-    </Svg>
-  );
-};
-
-export default DashboardNameSubHeader;
```
Comment (rohan-bes) [inline packages/lesmis/src/components/DashboardExportModal/components/DashboardNameSubHeader.js:16]: This file wasn't being used anywhere yeah?

**Review by rohan-bes — APPROVED** [review]
Nice one @billli0 !

File: packages/lesmis/src/components/DashboardExportModal/components/DashboardNameSubHeader.js (hunk 2 above)
Comment (billli0) [reply packages/lesmis/src/components/DashboardExportModal/components/DashboardNameSubHeader.js:16]: No it doesn't get used.

**Pull request merged**